src/main/java/randoop/main/Minimize.java

```diff
@@ -0,0 +1,1078 @@
+   * declaration statement is assumed to declare only one variable, for instance, {@code int i;}.
+   * Multiple variable declarations in a single statement in the form {@code int i, j, k;} are not
+   * valid.
+   *
+   * @param vdExpr variable declaration expression that represents the statement to simplify
+   * @return a {@code Statement} object that is equal to {@code vdExpr} without the assignment to
+   *     the declared variable. Returns {@code null} if more than one variable is declared in the
+   *     {@code VariableDeclarationExpr}.
+   */
+  private static Statement removeLeftHandSideSimplifiation(VariableDeclarationExpr vdExpr) {
+    // Copy the variable declaration expression.
+    VariableDeclarationExpr exprCopy = (VariableDeclarationExpr) vdExpr.clone();
+    List<VariableDeclarator> vars = exprCopy.getVars();
+    if (vars.size() > 1) {
+      // More than one variable declared in this expression.
+      return null;
+    }
+    VariableDeclarator vd = vars.get(0);
+
+    // Create a new statement with only the right hand side.
+    ExpressionStmt newStmt = new ExpressionStmt(vd.getInit());
+    exprCopy.setParentNode(newStmt);
+    return newStmt;
+  }
+
+  /**
+   * Simplify the variable type names in a compilation unit. For example, {@code java.lang.String}
+   * should be simplified to {@code String}.
+   *
+   * @param compUnit compilation unit containing an AST for a Java file, the compilation unit will
+   *     be modified if a correct minimization of the method is found
+   * @param packageName the name of the package that the Java file is in
+   * @param filePath absolute file path to the input Java file
+   * @param classpath classpath needed to compile and run the Java file
+   * @param expectedOutput expected standard output from running the JUnit test suite
+   * @param timeoutLimit number of seconds allowed for the whole test suite to be run
+   * @return {@code CompilationUnit} with fully-qualified type names simplified to simple type names
+   */
+  private static CompilationUnit simplifyVariableTypeNames(
+      CompilationUnit compUnit,
+      String packageName,
+      String filePath,
+      String classpath,
+      String expectedOutput,
+      int timeoutLimit) {
+    // Map from fully-qualified type name to simple type name.
+    Map<String, String> typeNameMap = new HashMap<String, String>();
+    // Set of fully qualified type names that are used in variable declarations
+    Set<Type> fullyQualifiedNames = new HashSet<Type>();
+    // Collect all of the type names in the compilation unit.
+    new TypeVisitor().visit(compUnit, fullyQualifiedNames);
+
+    // Iterate through the set of fully qualified names and fill the map
+    // with mappings from fully qualified names to simple type names. Also
+    // add in necessary import statements.
+    for (Type type : fullyQualifiedNames) {
+      String typeName = type.toString();
+      addImport(compUnit, typeName);
```

Comment:
It's possible that an incorrect import will still compile -- for example, the two different classes of the same name might have the same interface, such as when one of them is a mock.